packages/eds-core-react/src/components/Slider/Slider.stories.tsx

```diff
@@ -1,29 +1,67 @@
 import { useState, FormEvent } from 'react'
 import { Slider, SliderProps } from '../..'
-import { Story, Meta } from '@storybook/react'
+import { Story, ComponentMeta } from '@storybook/react'
+import page from './Slider.docs.mdx'
 
 export default {
   title: 'Inputs/Slider',
   component: Slider,
   parameters: {
     docs: {
-      description: {
-        component: `A slider is a flexible input that allows users to adjust
-        predefined values.
-        `,
-      },
+      page,
     },
   },
-} as Meta
+} as ComponentMeta<typeof Slider>
 
-export const Default: Story<SliderProps> = (args) => {
+export const Introduction: Story<SliderProps> = (args) => {
   return (
     <div style={{ margin: '3rem' }}>
       <Slider {...args} />
     </div>
   )
 }
 
+export const SimpleSlider: Story<SliderProps> = () => (
+  <>
+    <span id="simple-slider">Slide me</span>
+    <Slider value={4} min={0} max={10} ariaLabelledby="simple-slider" />
+  </>
+)
+SimpleSlider.storyName = 'Simple slider'
```

Comment:
The derived name from story should be the same as your are overriding here. So this should no longer be needed.

Reply:
I kept it for the menu purpose. It keeps the consistency with the rest headers in the menu.

Reply:
If I am not mistaken Storybook with translate the story name `SimpleSlider` to `Simple slider` so having the this defined is redundant no?

Reply:
With `storyName`:
<img width="112" alt="Screenshot 2022-04-20 at 11 50 38" src="https://user-images.githubusercontent.com/7075271/164201614-95e09d9e-33cb-4804-b5c3-f95ea615ecf4.png">

Without `storyName`: 
<img width="109" alt="Screenshot 2022-04-20 at 11 51 45" src="https://user-images.githubusercontent.com/7075271/164201807-af90c5d5-c5a6-468e-b50d-60b87ea72aee.png">

What do you prefer?
I kept in all the stories i worked on option with `storyName`, because sometimes there are longer  names: 
<img width="184" alt="Screenshot 2022-04-20 at 11 53 55" src="https://user-images.githubusercontent.com/7075271/164202273-148525ef-3c1f-437d-a08d-34cf924f7395.png">
<img width="194" alt="Screenshot 2022-04-20 at 11 55 30" src="https://user-images.githubusercontent.com/7075271/164202602-09e3a82b-adcf-40ff-952c-8049512ab8b1.png">

Reply:
Hmm, the "Simple slider" looks better I guess. Keep it as is then?  

I see we have a good mix of with and without higher/lower cases in our other stories. We need to change the others that are missing to match the style then :)

Reply:
OK, yes i keep in mind the naming on each element i work on.